host/utils/latency/lib/Responder.cpp

```diff
@@ -64,15 +64,13 @@ void Responder::FLUSH_SCREEN()
 // Like FLUSH_SCREEN but with new line
 void Responder::FLUSH_SCREEN_NL()
 {
-    do {
-        int y, x;
-        getyx(_window, y, x);
-        if (x > 0) {
-            printw("\n");
-            y++;
-        }
-        FLUSH_SCREEN();
-    } while (0);
+    int y, x;
```

Comment:
It would be nice if someone could take a closer look at this part. To me, this looked a bit off:

- It seems like this was copied out of some ancient MACRO because of the `do{} while(0)` usage
- The `y` variable is unused but was incremented. I am not sure what the original intent here was, so this might still not do what it is supposed to.